### DISCUSSION TITLE
Wrong day displayed for Sunday meetings

### DISCUSSION THREAD

**Discussion by DerekDen** [code4recovery/12-step-meeting-list#903] [Q&A] [answered]
I am seeing  some weird behaviour of the meeting list.   The meetings that are on Sunday display as being on Monday.  If I select to just show sunday meetings it shows the correct meetings but labels them as Monday

<img width="917" alt="Screenshot 2022-09-05 at 09 43 21" src="https://user-images.githubusercontent.com/112857352/188407798-f461d11c-560e-433e-ac92-5f12f531b1c8.png">

<img width="420" alt="Screenshot 2022-09-05 at 09 46 32" src="https://user-images.githubusercontent.com/112857352/188408202-c619e510-507b-42ba-af28-6c86395fca2d.png">

<img width="837" alt="Screenshot 2022-09-05 at 09 48 15" src="https://user-images.githubusercontent.com/112857352/188408540-1c90188d-2927-4d96-af03-09f65361093d.png">

The site is aaportugal.net

Any help/hints greatly appreciated.   Derek

**Comment by DerekDen** [comment]
The timezone is set correctly 
<img width="746" alt="Screenshot 2022-09-05 at 10 06 48" src="https://user-images.githubusercontent.com/112857352/188412500-17677ea2-a07c-46b6-8d0a-6b9e6535d16d.png">

**Reply by joshreisner** [reply]
thanks @DerekDen fix coming shortly

**Reply by joshreisner** [reply]
hi @DerekDen please give it a refresh and let me know if you're still seeing any weird behavior. 

<img width="1269" alt="Screen Shot 2022-09-05 at 5 14 00 AM" src="https://user-images.githubusercontent.com/1551689/188446842-3945e014-86fe-4316-9675-b2ca6741b94e.png">

**Reply by DerekDen** [reply]
Hi, thanks for your response.

I have been using Safari, refreshing makes no difference, but I've now used Chrome & Firefox and they seem fine.  I removed the cached data in Safari, and the list is still displayed incorrectly, note how the Sunday meetings say Monday

<img width="746" alt="Screenshot 2022-09-05 at 13 30 12" src="https://user-images.githubusercontent.com/112857352/188449803-8c02a485-1350-46db-9a79-bacea9037a64.png">

**Comment by DerekDen** [comment, marked as answer]
Restarting Safari after I cleared the cache was needed.  It's now fine.

**Reply by joshreisner** [reply]
oh great, glad to hear it.

sorry about that! let us know if you observe any more strange behavior

**Reply by DerekDen** [reply]
I don't think this is connected, but I'm seeing something odd in Safari and Firefox.  Our initial meeting list came from a TMSL export, but now I need to add a couple of meetings that were never listed at Region.   I create the meeting, then start adding the details... As soon as I enter the Zoom URL the Publish button Greys out, and there seems to be nothing I can do to get the ability to publish the meeting.  Am I missing something?

in fellowship, Derek

**Reply by joshreisner** [reply]
That does sound like a separate issue, would you mind creating  a new discussion topic for that?

Thanks

**Reply by DerekDen** [reply]
I think this has all been because of some strange caching at the hosting site.  I cleared all their caches, and all is now OK.
